CHANGELOG.md

```diff
@@ -16,6 +16,12 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 - **Breaking** Minimum supported Xcode version bumped to 12.2.
 
+## 1.45.2
+
+### Fixed
+
+- Fixed an issue with `tuist dependencies` command failed for `Carthage` dependencies imported as binaries [#3108] by [@havebeenfitz](https://github.com/havebeenfitz)
```

Comment:
I think it is more a "support for Carthage framework dependencies" rather than a fix

Reply:
[Please take a look at this comment)](https://github.com/tuist/tuist/pull/3164#issuecomment-876533830)